src/templates/resource-list/style.scss

```diff
@@ -45,10 +37,14 @@ ul.boxes {
 
 ul {
  padding: 0;
-   -webkit-column-count: 2;
-      -moz-column-count: 2;
-           column-count: 2;
-   -webkit-column-gap: 30px;
-      -moz-column-gap: 30px;
-           column-gap: 30px;
+ -webkit-column-count: 2;
```

Comment:
Not sure why the linter isn't picking this up, but we don't allow vendor prefixes and this block should be 2 spaces indented instead of 1.